### ISSUE TITLE
Implement regressive matching on url

### ISSUE THREAD

**Issue by jvoisin** [nbs-system/nxtool-ng#13]
It would be great to have regressive matching on url, like:
- `/pif/paf/pouf/test.gif`
- `/pif/paf/pouet`
- `/pif/pof` 

would result in a rule on `/pif/`.

**Comment by buixor** [comment]
You mean :
^/pif/[a-z/]+(\.gif)*$ ?
Generating regex without supervision sounds quite dangerous, as one might abuse the regex, with an url such as :

/pif/../../../another-file.php

**Comment by jvoisin** [comment]
I don't think that the infamous path traversal trick (`..`) is working on url anymore.

I do mean `^/pif/`.
